src/cmd/run.rs

```diff
@@ -104,11 +108,42 @@ pub async fn main<D: Database>(ctx: &AppContext, config: Config, db: D) -> Resul
         db,
     };
 
+    for integration in ctx.config.integrations().iter() {
+        if let Err(err) = validate_integration_configuration(&ctx.db, integration).await {
+            warn!("Unable to validate '{integration}': {err:#?}");
```

Comment:
What action can the user take here? Ensure things presented to the user are actionable and they're told what the next steps are (even if that action is simply "contact support").